src/underactuated/multibody.py

```diff
@@ -18,7 +20,11 @@ def ManipulatorDynamics(plant, q, v=None):
 
     else:
         # MultibodyPlant version
-        assert(isinstance(plant, MultibodyPlant))
+        T = None
+        for scalar in [float, AutoDiffXd, Expression]:
+            if isinstance(plant, MultibodyPlant_[scalar]):
+                T = scalar
```

Comment:
@EricCousineau-TRI -- is this the right way to infer the template type of an argument?  is there a better way?

Reply:
Seeing this now - you should also be able to do `MultibodyForce_(plant)`, which was enabled by this PR:
https://github.com/RobotLocomotion/drake/pull/11705